[FIX] website: Display correctly top buttons of the editor sidebar

nboulif opened the pull request:
Steps to reproduce :

  - Install website module
  - Activate (and translate website) French language
  - Go to translated terms and fetch for "Discard" in website module
  - Replace the translated value by "Ne pas sauvegarder"
  - Go to Website -> Configuration -> Websites
  - Select main website and change language to french then save
  - Go to Website and edit homepage
  - Add any block

Issues :

  On top of editor sidebar, buttons are not displayed correctly.

Solutions :

  Add css class `d-flex` to the div around the buttons.

opw-2683602

nboulif commented:
ping @rdeodoo

rdeodoo commented:
Quickly tested, this is not enough, it will fix the `Ne pas sauvegarder` but it is true until it isn't.. Add a few more char and it still break.
Add `d-flex` on `o_we_external_history_buttons` also and it should be good

nboulif commented:
Hello @rdeodoo , thanks for the review!
Indeed, my bad, I added a button "Preview" but did not make it longer. I updated the fix, thanks :+1: 
I also tried the use case where there is a mobile preview button
https://github.com/odoo/odoo/blob/b5f135f83e576f6b552392249913dacca45a7650/addons/web_editor/views/snippets.xml#L4-L8
1px too much on the left :sweat_smile: 
![image](https://user-images.githubusercontent.com/44211228/140907570-afad125a-8e4b-41c7-bce4-fb89cf6823b2.png)

Not sure if it must be fixed too, but I tried and did not find a good fix (except replacing the size of the padding and font size of the buttons)... any idea?

rdeodoo commented:
We don't have that button yet, isn't it? As this is only a 1px issue, we will fix it when we actually show that button, no need of premature optimisation, since that button my might even be there when we will use it.
@robodoo r+

rdeodoo commented:
For the record, same-ish issue as https://github.com/odoo/odoo/pull/62098
